java/clip-geometry/src/main/java/com/esri/arcgisruntime/sample/clipgeometry/MainActivity.java

```diff
@@ -101,6 +101,8 @@ protected void onCreate(Bundle savedInstanceState) {
           clipAreasOverlay.getGraphics().add(clippedGraphic);
         }
       }
+      // disable button
+      clipButton.setEnabled(false);
```

Comment:
Should you disable the button directly before performing the clip operation? The clip operations may not be instantaneous leaving a chance to click it again.